src/Data/API/Types.hs

```diff
@@ -296,6 +299,7 @@ instance NFData BasicType where
 -- | A default value for a field
 data DefaultValue
     = DefValList
+    | DefValSet
```

Comment:
I'm not sure we need `DefValSet` with its own syntax, can we just use `DefValList` for both?